Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "StipplePlotly"
 uuid = "ec984513-233d-481d-95b0-a3b58b97af2b"
 authors = ["Adrian <e@essenciary.com> and contributors"]
```

Comment:
Please add yourself as an author here if you want :-)

Reply:
OK, I will add my name.